internal/test/wmcb/wmcb_test.go

```diff
@@ -297,6 +328,48 @@ func (vm *wmcbVM) runTestKubeletUninstall(t *testing.T) {
 	require.NoError(t, err, "TestKubeletUninstall failed")
 }
 
+// configureContainerd installs the containerd Windows service
+func (vm *wmcbVM) configureContainerd() error {
+	// Check if containerd is running
+	out, err := vm.Run("Get-Process -Name \"containerd\"", true)
+
+	// stderr being empty implies that containerd was running.
+	if err == nil || out == "" {
```

Comment:
I am not clear on why the second clause `|| out == ""` is included in this check. What if the command results in an error without any output? I've seen the following for example:
```
error running	{"cmd": "powershell.exe -NonInteractive -ExecutionPolicy Bypass \"Test-Path C:\\k\\cni\\\\win-bridge.exe\"", "out": "", "error": "read tcp 192.168.81.68:36090->192.168.81.111:22: read: connection reset by peer"}
```

Reply:
Please make this a shared function with any Windows process instead of [having duplicate code](https://github.com/openshift/windows-machine-config-bootstrapper/blob/38466b767564bddeeb68a6a9fab708dd73a937ce/internal/test/wmcb/wmcb_test.go#L218-L224)

Reply:
yes initially thought of doing that, missed my later on. Thanks for pointing out the duplicity.

Reply:
@saifshaikh48 we were always using the same [condition](https://github.com/openshift/windows-machine-config-bootstrapper/blob/38466b767564bddeeb68a6a9fab708dd73a937ce/internal/test/wmcb/wmcb_test.go#L218-L224).
Looking at vm.Run() method, it only returns err with out  "" OR out with an err = nil.
I am trying to change the method to use sc.exe but running into some powershell issues, if it's a low hanging fruit I ll handle it.